lib/visualize_packs.rb

```diff
@@ -1,29 +1,215 @@
 # typed: strict
```

Comment:
I notice you're keeping the type sigil but dropping type check. This file drops sigs but still has the strict sigil, which means type checking (even at level false) will break for other devs. I'd be down to help fix the type check, even if that makes lowering the type sigil if we're not getting value from the types.

Reply:
I will remove for now.